client/src/components/Applications/AppLayout/AppLayout.js

```diff
@@ -15,64 +15,78 @@ const AppsLayout = appDict => {
     }),
     shallowEqual
   );
-  const appMeta = appDict[params.appId];
+  const hasCategoryDict = Boolean(Object.keys(categoryDict).length);
 
   return (
     <>
-      <div className="apps-header">
-        <h5>
-          Applications
-          {appMeta ? ` / ${appMeta.value.definition.label}` : ''}
-        </h5>
-      </div>
-      {loading && !Object.keys(categoryDict).length ? (
+      {loading && !hasCategoryDict ? (
         <LoadingSpinner />
       ) : (
         <>
-          {Boolean(Object.keys(categoryDict).length) && <AppBrowser />}
-          {!params.appId && (
-            <AppPlaceholder apps={Boolean(Object.keys(categoryDict).length)} />
-          )}
+          {hasCategoryDict && <AppBrowser />}
+          {!params.appId && <AppPlaceholder apps={hasCategoryDict} />}
         </>
       )}
     </>
   );
```

Comment:
- Replace `Object.keys(categoryDict).length` and `Boolean(Object.keys(categoryDict).length)` with `hasCategoryDict` to:
    - Have consistent value types.
    - Abstract the logic that sets the value from the use of the value.
    - Allow me to initially follow the code more easily.
- Move `appMeta` from `AppsLayout` to new `AppsHeader`.